src/components/Feed/Feed.css

```diff
@@ -13,10 +13,22 @@ p {
 }
 
 header {
+    position: fixed;
+    width: 100%;
+    right: 0;
+    top: 0;
     display: flex;
-    justify-content: space-between;
-    padding: 1rem;
+    flex-shrink: 1;
+    z-index: 10;
+    padding: 1rem 1rem 1rem 24rem;
     background: transparent;
+    box-sizing: border-box;
+}
+
+.flex-container {
```

Comment:
I'm not sure that splitting this up is totally necessary. 


I would move the display: flex back into the header component and remove the flex-container div in the Feed component